### DISCUSSION TITLE
Enter URL? What should be the path?

### DISCUSSION THREAD

**Discussion by JoshFink** [arabcoders/watchstate#282] [General] [answered]
Hi, probably really simple but I'm trying to figure out what the path should be when it asks when adding a backend:

Enter [plex] URL

Thanks

**Comment by arabcoders** [comment, marked as answer]
Hi, usually depends on how you exposing plex, but if it's not in docker you could try `http://localhost:32400`.

You can also use this command 

```shell
$ docker exec -ti watchstate console plex:discover  -- 'your_plex_token'
```

to see list of servers and URLs that are associated with given plex token.

**Comment by JoshFink** [comment]
Thank you. I'll try that out. I appreciate the quick help
